optimism/src/mips/witness.rs

```diff
@@ -299,18 +308,30 @@ impl<Fp: Field> Env<Fp> {
 
     pub fn step(&mut self, config: VmConfiguration, metadata: &Meta, start: &Start) {
```

Comment:
Are instruction parts the only witness values that need to be computed for each executed instruction, or will there be more in subsequent PRs?